### PR TITLE
connect routes and complete the profile page frontend

### PR DESCRIPTION
**Pull request by KimlengHor**
Accomplish:
1. Create a route for each page.  Connect between the dashboard and the profile page.
2. Complete profile page.

### CODE REVIEW

File: client/src/Profile.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+                        <IconButton onClick= {handleEdit} id="editIcon">
+                            <CreateIcon></CreateIcon>
+                        </IconButton>
+
+                        <IconButton onClick= {handleEdit} id="checkIcon">
+                            <CheckIcon></CheckIcon>
+                        </IconButton>
+
+                        <label id="editLabel">Edit</label>
```
Comment (dpham123) [inline client/src/Profile.js]: Can we get `Personal Information` to look something like this:
image

The edit button is fine. I would just prefer the text and button to be all on the same line with a divider underneath.

Comment (juliachin123) [reply client/src/Profile.js]: Agreed.

Comment (KimlengHor) [reply client/src/Profile.js]: I just made some changes and add a new page (personal recipe)

File: client/src/Profile.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    button: {
+        margin: theme.spacing(3, 0, 2),
+    },
+    form: {
+        width: '100%', // Fix IE 11 issue.
+        marginTop: theme.spacing(1),
+    },
+}));
+
+const handleEdit = () => {
```
Comment (dpham123) [inline client/src/Profile.js]: The `Edit` icon looks good. Is it possible to change the text from gray to black when clicking the `Edit` button?

File: client/src/Preference.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            <Grid item xs={12} sm={8} md={5} component={Paper} elevation={6} square>
+                <div className={classes.paper}>
+                    <label id="question">What type of food do you like?</label>
+
+                    <Grid className={classes.button}>
+                        <ButtonGroup fullWidth aria-label="full width outlined button group">
+                            <Button>Asian</Button>
+                            <Button>African</Button>
+                            <Button>European</Button>
+                        </ButtonGroup>
```
Comment (dpham123) [inline client/src/Preference.js]: I'm assuming you will work on changing the button colors when they are selected in a future PR?

File: client/src/PersonalRecipe.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      { title: 'Food', field: 'name' },
+      // { title: 'Price', field: 'birthYear', type: 'numeric' },
+    ],
+    data: [
+      { name: 'Banh mi', birthYear: 1987 },
+      {
+        name: 'Pho',
+        birthYear: 2017
+      },
+    ],
```
Comment (dpham123) [inline client/src/PersonalRecipe.js]: I'm assuming you will work on this more in a future PR? If so, maybe we can add images to the side?

File: client/src/Allergy.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+                    <label id="question">Do you have any food allergies?</label>
+
+                    <Grid className={classes.button}>
+                        <div className = "firstAllergySelection">
+                            <ButtonGroup fullwidth aria-label="full width outlined button group" id="firstAllergyButtons">
+                                <Button>Eggs</Button>
+                                <Button>TreeNuts</Button>
+                                <Button>Peanuts</Button>
+                                <Button>Shellfish</Button>  
+                            </ButtonGroup>
```
Comment (dpham123) [inline client/src/Allergy.js]: I'm assuming you will work on changing the button colors when they are selected in a future PR?

File: client/src/Profile.js [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+                        />
+
+                        <TextField
+                            id="password"
+                            label="Password"
+                            margin="normal"
+                            fullWidth
+                            type='password'
+                            disabled = 'true'
+                        />
```
Comment (dpham123) [inline client/src/Profile.js]: Can we move the password to another section, kind of like this:
image

Comment (KimlengHor) [reply client/src/Profile.js]: okay sounds good

File: client/src/Allergy.js (hunk 5 above)
Comment (KimlengHor) [reply client/src/Allergy.js]: yes

File: client/src/PersonalRecipe.js (hunk 4 above)
Comment (KimlengHor) [reply client/src/PersonalRecipe.js]: yes

File: client/src/Preference.js (hunk 3 above)
Comment (KimlengHor) [reply client/src/Preference.js]: yes

File: client/src/Profile.js (hunk 2 above)
Comment (KimlengHor) [reply client/src/Profile.js]: yes sir

**Pull request merged**